images/scripts/fedora-35.bootstrap

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+#
+# Copyright (C) 2021 Red Hat Inc.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+# 02110-1301 USA.
+
+set -eux
+
+URL='https://download.fedoraproject.org/pub/fedora/linux/releases/35/Cloud/x86_64/images/'
```

Comment:
For the record, this url won't exist for some time, first it will live under https://download.fedoraproject.org/pub/fedora/linux/development/35/Cloud/x86_64/images/

Reply:
ha, and there are even composes already!

Reply:
Ah, I see.